runtime/service_integ_test.go

```diff
@@ -265,7 +263,9 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 	// container ends up in the right VM by assigning each VM a network device with a unique mac address and having each container
 	// print the mac address it sees inside its VM.
 	var vmWg sync.WaitGroup
-	for vmID, c := range cases {
+	for i := 0; i < numberOfVms; i++ {
+		caseTypeNumber := 1 % 2
```

Comment:
this should be `caseTypeNumber := 1 % len(cases)`

Reply:
If the intention is to switch back and forth between test cases (which I think is good), then I would have thought it should be `i % len(cases)`

Reply:
Actually it doesn't matter in this particular case, coz we have only two cases, and it will be or 0 or 1, anyway, but I agree that in case we will have more than 2 cases, it make sense to use `i % len(cases)`

Reply:
Sorry, yes, I meant i % len(cases). This is easier to maintain and doesnt require us to change anything if we add more cases in the future.